.circleci/config.yml

```diff
@@ -15,27 +15,23 @@ jobs:
       - run:
           command: ./manage.py test
           name: Test
-      - run:
-           name: Deploy Master to Heroku
-           command: |
-             git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
 
-  deploy:
+  deployment:
     machine:
       enabled: true
     steps:
       - checkout
       - run:
            name: Deploy Master to Heroku
            command: |
-             git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
-
-
```

Comment:
In master branch also these changes are present but it is still showing these changes, I can't understand why?
https://github.com/SourabhTestpress/NewBoardApp/blob/master/.circleci/config.yml

Reply:
Rebase this branch properly. Look into `git rebase -i`